apps/src/templates/VersionRow.jsx

```diff
@@ -21,58 +26,79 @@ export default class VersionRow extends React.Component {
     return timestamp.toString();
   }
 
+  getUrlAttributes() {
+    const userId = queryParams('user_id');
+    const viewAs = queryParams('viewAs');
+    const params = {};
+    if (viewAs) {
+      params.viewAs = viewAs;
+    }
+    if (userId) {
+      params.user_id = userId;
+    }
+    if (!this.props.isLatest) {
+      params.version = this.props.versionId;
+    }
+    return queryString.stringify(params);
```

Comment:
The ifs could probably also be written by
```
const params = {
  viewAs: queryParams('viewAs') || "",
  user_id: queryParams('user_id') || "",
  version: this.props.isLatest ? "" : this.props.versionId
  }
  but actually I think what you have here is pretty clear!!

Reply:
Thanks! I think this wouldn't quite work as-is because I only want the params in the url if they aren't blank